sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -33,7 +33,7 @@
 class StorageClientTest(StorageTestCase):
     def setUp(self):
         super(StorageClientTest, self).setUp()
-        self.sas_token = '?sv=2015-04-05&st=2015-04-29T22%3A18%3A26Z&se=2015-04-30T02%3A23%3A26Z&sr=b&sp=rw&sip=168.1.5.60-168.1.5.70&spr=https&sig=Z%2FRHIX5Xcg0Mq2rqI3OlWTjEg2tYkboXr1P9ZUXDtkk%3D'
+        self.sas_token = '?sv=2015-04-05&st=2015-04-29T22%3A18%3A26Z&se=2015-04-30T02%3A23%3A26Z&sr=b&sp=rw&sip=168.1.5.60-168.1.5.70&spr=https&sig=Y%2FoUshaLLnoTPass'
```

Comment:
Do we need to have any of the Sas_Token in code?  I'm not a huge fan of the "shallnotpass" text...kind of mocking tone.   Can we just do:

`self.sas_token = ''`

And then inject the new one at runtime?

This comment applies to all sas_tokens in this PR.